student.rb

```diff
@@ -1,9 +1,11 @@
 require_relative 'person'
 
 class Student < Person
```

Comment:
- [ ] The Student should have a method for the association.

  ```
  def classroom=(classroom)
    @classroom = classroom
    classroom.students.push(self) unless classroom.students.include?(self)
  end
  ```

Reply:
done